songr/src/main/resources/templates/albums.html

```diff
@@ -29,6 +30,33 @@ <h1>My Personal Jams</h1>
         </tr>
         </tbody>
     </table>
+    <form action = "/newAlbum" method="POST">
+        <fieldset class="outerFieldset">
+            <legend class="formTitle"><h1>Add New Albums</h1></legend>
+        <fieldset id="top" class="inputContainer">
+            <label>Album Title: </label>
+            <input type="text" name="title">
+        </fieldset>
+        <fieldset class="inputContainer">
+            <label>Artist Name: </label>
+            <input type="text" name="artist">
+        </fieldset>
+        <fieldset class="inputContainer">
+            <label>Song Count: </label>
+            <input type="text" name="songCount">
```

Comment:
I recommend making this and the length inputs `type="number"` so nobody can accidentally enter text. Not a requirement, but it'll make your life easier when you're testing :)